price-estimator/docker/deploy.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+image_name=$1
+
+echo "Docker login"
+echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin;
+
+echo "Building price estimator docker";
+docker build --tag gnosispm/dex-price-estimator-rust:$image_name -f price-estimator/docker/Dockerfile .
+
+echo "Pushing price estimator docker"
+docker push gnosispm/dex-price-estimator-rust:$image_name;
+
+echo "done"
+
+if [ "$image_name" == "master" ] && [ -n "$AUTODEPLOY_URL_PRICE_ESTIMATOR" ] && [ -n "$AUTODEPLOY_TOKEN_PRICE_ESTIMATOR" ]; then
```

Comment:
I don't think the token is needed anymore. @davidalbela can you confirm?

Reply:
It's set to "notrequired" in CI so probably not but we are using it in the script in the nodejs price estimator even though it has this value.